Makefile

```diff
@@ -5,17 +5,25 @@ MKDOCS_VER = 8.3.9
 # make sure to also change the mkdocs version in actions' cicd.yml and force-build.yml files
 MKDOCS_INS_VER = 8.5.11-insiders-4.27.0-hellt
 
+DATE := $(shell date)
+COMMIT_HASH := $(shell git rev-parse --short HEAD)
+
+LDFLAGS := -s -w
+LDFLAGS += -X 'github.com/srl-labs/containerlab/cmd.version=0.0.0'
```

Comment:
Hi @oshothebig 
Why don't we declare this variable once with all the flags in it?

Reply:
I'll modify to put all the flags into a single variable.

Reply:
The variable declaration is revised.